src/main/java/duke/EmptyDescription.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+/**
+ * Exception when no description is found.
+ */
+class EmptyDescription extends DukeException {
+    /**
+     * Constructor for the EmptyDescription class.
+     */
+    public EmptyDescription() {
```

Comment:
Nice abstraction for `EmptyDescription` exceptions! Perhaps you can also include to use this `EmptyDescription` for other tasks (and not limited to toDos.